master-server/src/main/java/com/replicated_log/master_server/service/AckService.java

```diff
@@ -0,0 +1,40 @@
+package com.replicated_log.master_server.service;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class AckService {
+
+    private static int secondariesNumber;
+    private static int acksReceived;
```

Comment:
static filed should be as final fields. Static is a contstant, so should be with upper_case field naming

Reply:
in iteration 1 these fields can change their values, so they are not constants, that`s why camel case naming was used for these fields.